From 5ca8fdcb62fe4ade011b22f9d0ceba16aab2bce7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Wed, 15 Nov 2023 15:21:26 +0100
Subject: [PATCH] hwdb: rename .html=>.csv

The old suffix is now confusing.

(cherry picked from commit 94113d5f19e5768c206176684e63525f679988d3)

Related: RHEL-5950
---
 hwdb.d/acpi-update.py                                  | 4 ++--
 hwdb.d/{acpi_id_registry.html => acpi_id_registry.csv} | 0
 hwdb.d/{pnp_id_registry.html => pnp_id_registry.csv}   | 0
 tools/update-hwdb.sh                                   | 4 ++--
 4 files changed, 4 insertions(+), 4 deletions(-)
 rename hwdb.d/{acpi_id_registry.html => acpi_id_registry.csv} (100%)
 rename hwdb.d/{pnp_id_registry.html => pnp_id_registry.csv} (100%)

diff --git a/hwdb.d/acpi-update.py b/hwdb.d/acpi-update.py
index f65147c91d..386575067b 100755
--- a/hwdb.d/acpi-update.py
+++ b/hwdb.d/acpi-update.py
@@ -28,5 +28,5 @@ print('# This file is part of systemd.\n'
       '#     https://uefi.org/uefi-pnp-export\n'
       '#     https://uefi.org/uefi-acpi-export')
 
-read_table("acpi_id_registry.html")
-read_table("pnp_id_registry.html")
+read_table('acpi_id_registry.csv')
+read_table('pnp_id_registry.csv')
diff --git a/hwdb.d/acpi_id_registry.html b/hwdb.d/acpi_id_registry.csv
similarity index 100%
rename from hwdb.d/acpi_id_registry.html
rename to hwdb.d/acpi_id_registry.csv
diff --git a/hwdb.d/pnp_id_registry.html b/hwdb.d/pnp_id_registry.csv
similarity index 100%
rename from hwdb.d/pnp_id_registry.html
rename to hwdb.d/pnp_id_registry.csv
diff --git a/tools/update-hwdb.sh b/tools/update-hwdb.sh
index abbbb82f4b..096a10016b 100755
--- a/tools/update-hwdb.sh
+++ b/tools/update-hwdb.sh
@@ -21,8 +21,8 @@ if [ "${2:-}" != "-n" ]; then (
     curl --fail -L -o ma-large.txt 'http://standards-oui.ieee.org/oui/oui.txt'
     curl --fail -L -o ma-medium.txt 'http://standards-oui.ieee.org/oui28/mam.txt'
     curl --fail -L -o ma-small.txt 'http://standards-oui.ieee.org/oui36/oui36.txt'
-    curl --fail -L -o pnp_id_registry.html 'https://uefi.org/uefi-pnp-export'
-    curl --fail -L -o acpi_id_registry.html 'https://uefi.org/uefi-acpi-export'
+    curl --fail -L -o pnp_id_registry.csv 'https://uefi.org/uefi-pnp-export'
+    curl --fail -L -o acpi_id_registry.csv 'https://uefi.org/uefi-acpi-export'
 ) fi
 
 set -x
